src/Umbraco.Web.UI.Client/src/common/directives/components/tree/umbtreeitem.directive.js

```diff
@@ -115,6 +121,11 @@ angular.module("umbraco.directives")
                 }, 0, false);
             }
 
+            scope.isRtl = function () {

```

Comment:
Can you use the `isRtl` function here? Maybe in the `IsRtl` function update the scope? Not sure if that's ideal..